applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -956,16 +956,16 @@ int slm_at_host_init(void)
 
 	/* Initialize the UART module */
 #if defined(CONFIG_SLM_CONNECT_UART_0)
-	uart_dev = device_get_binding(DT_LABEL(DT_NODELABEL(uart0)));
+	uart_dev = DEVICE_DT_GET(DT_NODELABEL(uart0));
```

Comment:
Why not initialize it at build time? `uart_dev` could be `const` then.

Reply:
Can share piece of sample code?

Reply:
I mean something like:
```
#if defined(CONFIG_SLM_CONNECT_UART_0)
#define UART_NODE DT_NODELABEL(uart0)
#elif defined(CONFIG_SLM_CONNECT_UART_2)
#define UART_NODE DT_NODELABEL(uart2)
#else
#error "Unsupported UART instance"
#endif
static const struct device *const uart_dev = DEVICE_DT_GET(UART_NODE);
```

Reply:
Looks nice, thanks!
Please add by this PR.